platforms/Windows/installer-arm64.wxs

```diff
@@ -0,0 +1,25 @@
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:bal="http://wixtoolset.org/schemas/v4/wxs/bal">
+  <Bundle Name="Swift Developer Package for Windows aarch64" Version="$(var.ProductVersion)" Manufacturer="swift.org" UpgradeCode="151d42d9-8877-4b72-ac62-4695243a35c1">
```

Comment:
I think the `UpgradeCode` is the one Guid that we care about being unique per platform.

Reply:
Yeah, it is unique in this case.

Reply:
Oh my bad, I thought I had seen it being the same as amd64

Reply:
FTR (and because you scared me and made me check it again!) The X64 installer upgrade code is 8c75f32a-7bdf-4c61-abf6-c7e1c4b8fbf6.